### DISCUSSION TITLE
Welcome to PyNifly Discussions!

### DISCUSSION THREAD

**Discussion by BadDogSkyrim** [BadDogSkyrim/PyNifly#148] [Announcements]
## 👋 Welcome!
 I've got some changes planned and I thought I'd try using these discussions as a way to hold my thoughts and give folks a heads-up. Feedback is welcome.

I also may move some issues here if they turn into a discussion of proper behavior.

**Comment by podvornyakva** [comment]
Go on, Man. You are gorgeous.
